File: listviewadapter/app/src/main/java/group/learn/listviewadapter/MainActivity.java
package group.learn.listviewadapter;

import android.content.Intent;
import android.content.IntentFilter;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import group.learn.listviewadapter.components.MusicPlayerActivity;
import group.learn.listviewadapter.components.broadcast.AirplaneModeReceiver;
import group.learn.listviewadapter.components.broadcast.SpeakerDownReceiver;
import group.learn.listviewadapter.views.basicadapter.BuiltInAdapterView;
import group.learn.listviewadapter.views.customadapter.CustomAdapterView;
import group.learn.listviewadapter.views.gridview.GridActivity;
import group.learn.listviewadapter.views.planetapps.PlanetActivity;
import group.learn.listviewadapter.views.recycleview.RecycleActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        IntentFilter intentFilter = new IntentFilter("android.intent.action.AIRPLANE_MODE");
        IntentFilter intentFilter2 = new IntentFilter("android.media.VOLUME_CHANGED_ACTION");
        AirplaneModeReceiver br = new AirplaneModeReceiver();
        SpeakerDownReceiver br2 = new SpeakerDownReceiver();
        registerReceiver(br, intentFilter);
        registerReceiver(br2, intentFilter2);

        final Button buttonNavigateListView = findViewById(R.id.btnNavigateListView);
        final Button buttonNavigateCustomListView = findViewById(R.id.btnNavigateCustomListView);
        final Button buttonNavigateToPlanetView = findViewById(R.id.btnNavigateToPlanet);
        final Button buttonNavigateToGridViews = findViewById(R.id.btnNavigateToGrid);
        final Button buttonNavigateToRecycleView = findViewById(R.id.btnNavigateToRecycle);
        final Button buttonNavigateToMusicPlayer = findViewById(R.id.btnNavigateToMusicPlayer);

        doNavigate(buttonNavigateListView, new Intent(this, BuiltInAdapterView.class));
        doNavigate(buttonNavigateCustomListView, new Intent(this, CustomAdapterView.class));
        doNavigate(buttonNavigateToPlanetView, new Intent(this, PlanetActivity.class));
        doNavigate(buttonNavigateToGridViews, new Intent(this, GridActivity.class));
        doNavigate(buttonNavigateToRecycleView, new Intent(this, RecycleActivity.class));
        doNavigate(buttonNavigateToMusicPlayer, new Intent(this, MusicPlayerActivity.class));
    }

    private void doNavigate(Button buttonNavigateListView, Intent intentToTargetActivity) {
        buttonNavigateListView.setOnClickListener(v -> startActivity(intentToTargetActivity));
    }
}

File: 00-old-explore/11_VideoPlayer/app/src/main/java/com/java/a11_videoplayer/MainActivity.kt
package com.java.a11_videoplayer

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.provider.MediaStore.Images.Media
import android.widget.Button
import android.widget.MediaController
import android.widget.VideoView

class MainActivity : AppCompatActivity() {
    private lateinit var videoView: VideoView
    private lateinit var button: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        videoView = findViewById(R.id.videoView1)
        button = findViewById(R.id.buttonMove)
        videoView.setVideoPath(
            "android.resource://${packageName}/${R.raw.video}"
        )
        var mc : MediaController = MediaController(this)
        mc.setAnchorView(videoView)
        videoView.setMediaController(mc)


        button.setOnClickListener {
            var i : Intent
            i = Intent(this, MainActivity2::class.java)
            startActivity(i)
        }
    }
}